README.md

```diff
@@ -1,75 +1,46 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Hello Microverse!
 
-> One paragraph statement about the project.
+> A program sending greeings to microverse
```

Comment:
- [ ] I would suggest you add a blank space after this line so that your README file looks nicer. Right now the paragraph and the screenshot are clustered.